trade-finance-manager-ui/server/controllers/utilisation-reports/add-to-an-existing-payment/get-link-to-premium-payments-tab.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Maximum recommended URL length for security and performance reasons.
+ * @see {@link https://stackoverflow.com/a/48230425}
+ */
+const MAX_URL_CHARACTERS = 2048;
+
+/**
+ * Estimated character allowance for the base URL (without query parameters).
+ */
+const BASE_URL_CHARACTER_ALLOWANCE = 150;
+
+/**
+ * Maximum length for URL with query parameters.
+ * This allows for approximately 189 seven-digit IDs to be passed as query parameters (see below for calculation).
+ * Calculation: (2048 - 150) = 1898 chars, 1898 / (7 digits per ID + 3 chars for percent-encoded comma) ≈ 189
+ */
+const URL_WITH_PARAMS_MAX_LENGTH = MAX_URL_CHARACTERS - BASE_URL_CHARACTER_ALLOWANCE;
```

Comment:
@abhi-markan What are your thoughts on the above constants please?

I think a single constant without the calculation would be simpler/clearer, e.g:

```ts
URL_WITH_PARAMS_MAX_LENGTH = 189
```

This could be applied in other areas/microservices so would like a standardised definition to be agreed upon.

Reply:
Thanks @abhi-markan and agree RE centralised constants - I mentioned in the next comment, will resolve that comment so we just have this one 👍